haystack/document_store/base.py

```diff
@@ -125,7 +127,7 @@ def get_document_count(self, filters: Optional[Dict[str, List[str]]] = None, ind
 
     @abstractmethod
     def query_by_embedding(self,
-                           query_emb: List[float],
+                           query_emb: np.ndarray,
```

Comment:
Is this really a numpy array here or a List[float]?

Reply:
It should be a NumPy array here.